op-chain-ops/deployer/opsm/superchain.go

```diff
@@ -54,46 +52,13 @@ type DeploySuperchainOpts struct {
 	Logger      log.Logger
 }
 
-func DeploySuperchainForge(ctx context.Context, opts DeploySuperchainOpts) (DeploySuperchainOutput, error) {
+func DeploySuperchain(h *script.Host, input DeploySuperchainInput) (DeploySuperchainOutput, error) {
```

Comment:
Thoughts about returning a pointer to the deploy output, rather than copying the large struct?

Reply:
I like copying them because it avoids any chance of accidentally mutating fields. I tried to keep all the methods in `opsm` returning values for this reason.

Reply:
Happy to change this if you feel strongly.